sklearn/preprocessing/_encoders.py

```diff
@@ -356,6 +462,233 @@ def _compute_drop_idx(self):
                              for (val, cat_list) in
                              zip(self.drop, self.categories_)], dtype=np.int_)
 
+    @property
+    def _infrequent_enabled(self):
+        """Infrequent category is enabled."""
+        return (self.max_categories is not None and self.max_categories > 1 or
+                (isinstance(self.min_frequency, numbers.Integral)
+                    and self.min_frequency > 1) or
```

Comment:
> 1 or >=1

Reply:
Currently, if `min_frequency` is one and an int (the default), and by the current description:

> If int, categories with a smaller cardinality will be considered infrequent.

Then there will be no infrequent categories.

Reply:
oh right thanks for clarifying